.github/workflows/check-active-prs-and-slots.yml

```diff
@@ -0,0 +1,103 @@
+
+on:
+  schedule:
+    # Monday at 12 PM UTC - https://cron.help/#0_12_*_*_MON
+    - cron: "0 12 * * MON"
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+
+env:
+  GH_TOKEN: ${{ github.token }}
+
+defaults:
+  run:
+    shell: pwsh
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  check-pr-slots:
+    runs-on: ubuntu-latest
+    outputs:
+      slotIDs: ${{ steps.listOfSlots.outputs.slotIDs }}
+      slotsNeedDeletion: ${{steps.comparision.outputs.slotsNeedDeletion }}
+
+    steps:
+      - name: Checking out
+        uses: actions/checkout@v2
+
+      - name: Load .env file
+        uses: xom9ikk/dotenv@v2
+        with:
+          path: ./.github
+
+      - name: Azure CLI - Login
+        uses: azure/login@v1
+        with:
+          client-id: ${{ secrets.AZURE_CLIENT_ID }}
+          tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+          subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+
+      - name: Get list of deployed Slots
+        id: slotList
+        run: |
+          $slots = $(az webapp deployment slot list `
+            --name ${{ env.APP_SERVICE_NAME }} `
+            --resource-group ${{env.AZURE_RESOURCE_GROUP }} `
+            --query '[].name' `
+            --output tsv | `
+            sed 's/pr-//g')
+          echo "slots=$slots" >> $env:GITHUB_OUTPUT
+
+      - name: Get list of active PRs
+        id: prList
+        run: |
+          $active_prs=$(gh pr list --state open | awk '{print $1}')
+          echo "active_prs=$active_prs" >> $env:GITHUB_OUTPUT
+
+      - name: Compare PRs with Slots
+        id: comparision
+        run: |
+          # Comparing the number of Slots and PRs
+          $PRList =  "${{ steps.PRList.outputs.active_prs }}" -split ' '
+          $SlotList = "${{ steps.slotList.outputs.slots }}" -split ' '
+
+          $slotsExistThatRequireDeletion = $SlotList | Where-Object { $_ -notin $PRList }
+          # $slotsExistThatRequireDeletion ='111 112'
+          $slotsNeedDeletion = $slotsExistThatRequireDeletion.Length -gt 0
+
+          if ( ! $slotsNeedDeletion ) {
+            echo "✅ - Number of slots are equal to number of active PRs - 🏃 Skipping next step"
+          }
+          else {
+            echo "❌ - Number of slots are not equal to number of active PRs"
+          }
+          echo "slotsExistThatRequireDeletion=$slotsExistThatRequireDeletion" >> $env:GITHUB_OUTPUT
+          echo "slotsNeedDeletion=$slotsNeedDeletion" >> $env:GITHUB_OUTPUT
+
+      - name: List of slots that need to be deleted
+        if: ${{ steps.comparision.outputs.slotsNeedDeletion == 'True' }}
+        id: listOfSlots
+        run: |
+          # Slots that need to be deleted
+          $slotsExistThatRequireDeletion = ${{steps.comparision.outputs.slotsExistThatRequireDeletion }}
+          Write-Host "⚡- These slots need to be deleted : $slotsExistThatRequireDeletion"
+          echo "slotIDs=$slotsExistThatRequireDeletion" >> $env:GITHUB_OUTPUT
```

Comment:
this looks superfluous as it isn't modifying any content

Reply:
Yes, I just removed it